### PR TITLE
deps: upgrading GraalVM version to 21.3

### PR DESCRIPTION
**Pull request by suztomo**
Corresponding upgrade in native-image-support-java: https://github.com/GoogleCloudPlatform/native-image-support-java/pull/249

### CODE REVIEW

**Comment by suztomo** [comment]
"java-presubmit (cloud-devrel-kokoro-resources)" failed:

```
Already have image (with digest): gcr.io/cloud-builders/docker
Sending build context to Docker daemon  4.608kB
Step 1/7 : FROM ghcr.io/graalvm/graalvm-ce:ol7-java11-21.3.0
manifest unknown
```

https://pantheon.corp.google.com/cloud-build/builds/6e2e1a46-7454-4693-96f5-43f6517110e2;step=6?project=cloud-devrel-kokoro-resources

I guess my change was pointing to a non-existent tag.

Screen Shot 2021-11-02 at 3 45 47 PM

File: java/graalvm/Dockerfile [hunk 1]
```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM ghcr.io/graalvm/graalvm-ce:ol7-java11-21.2.0
+FROM ghcr.io/graalvm/graalvm-ce:java11-21.3.0
```
Comment (suztomo) [inline java/graalvm/Dockerfile:15]: Initially, I tried "ol7-java11-21.3.0" but that tag doesn't exist. Now using "java11-21.3.0".

Comment (dzou) [reply java/graalvm/Dockerfile:15]: This should work: `ghcr.io/graalvm/graalvm-ce:ol7-21.3`

Source: https://github.com/graalvm/container/pkgs/container/graalvm-ce

Comment (chanseokoh) [reply java/graalvm/Dockerfile]: > ghcr.io/graalvm/graalvm-ce:ol7-21.3

Note, this is Java 17.

Comment (suztomo) [reply java/graalvm/Dockerfile]: Updated to "ol7-java11-21.3". dzou@ told me that this "ol" stands for Oracle Linux and is required to setup gcloud tool.

Comment (chanseokoh) [reply java/graalvm/Dockerfile]: Well, the funny thing is, `:java11-21.3.0` is still Oracle Linux. A newer version actually.

**Comment by suztomo** [comment]
"java-presubmit (cloud-devrel-kokoro-resources)" succeeded.

**Pull request closed without merging**